.travis.yml

```diff
@@ -11,6 +11,7 @@ notifications:
   email: false
 install: true
 before_install:
+- chmod +x /tools/travis/build.sh
```

Comment:
I think it will be better if you change the execution mode locally then push it. In addition, I think the path to build.sh is wrong, it should be `./tools/travis/build.sh`